tests/Controller/EnMarche/HomeControllerTest.php

```diff
@@ -55,10 +59,9 @@ public function provideSitemaps()
             ['/sitemap.xml'],
             ['/sitemap_main_1.xml'],
             ['/sitemap_content_1.xml'],
-            ['/sitemap_committees_1.xml'],
-            ['/sitemap_events_1.xml'],
+            ['/sitemap_committees_1.xml', '/sitemap.xml'],
+            ['/sitemap_events_1.xml', '/sitemap.xml'],
```

Comment:
in order to cover more test cases, what about :
- loading Events data
- asserting `/sitemap_events_1.xml` returns a 200 with actuel content
- asserting `/sitemap_events_N.xml` returns a redirection to `sitemap.xml` (with N depending on fixtures size)
?